configure

```diff
@@ -4144,21 +4144,31 @@ fi
 # Setup SDL specifics for SDL based backends
 #
 if test "$_sdl" = auto ; then
-	find_sdlconfig
+	if test "$_pkg_config" = "yes" && $_pkgconfig --exists sdl3; then
```

Comment:
At this moment in time, I'd still put SDL3 as the latest option, e.g. SDL2 -> SDL1 -> SDL3, so it for sure does not break ports before the backend is thoroughly tested.